docs/readmes/lte/deploy_install.md

```diff
@@ -1,42 +1,40 @@
 ---
 id: deploy_install
-title: Install AGW on Debian Stretch.
+title: Install AGW
 hide_title: true
 ---
 
-For Installing AGW on Ubuntu refer to : [_deploy_on_ubuntu_](https://magma.github.io/magma/docs/lte/deploy_install_ubuntu)
-# Install Access Gateway on Debian Stretch (Bare Metal)
-
-### AGW deployment on Debian distribution is DEPRECATED! Release 1.5 is last release supporting debian based AGW.
+# Install Access Gateway on Ubuntu (Bare Metal)
 
 ## Prerequisites
 
 To setup a Magma Access Gateway, you will need a machine that
 satisfies the following requirements:
 
-- AGW_HOST: 64bit-X86 machine, hardware strongly recommended (not virtualized).
+- AGW_HOST: 64bit-X86 machine, baremetal strongly recommended (not virtualized).
   You will need two ethernet ports. We use enp1s0 and enp2s0 in this guide.
   They might have different names on your hardware so just replace enp1s0 and
   enp2s0 with your current interfaces name in this guideline.
   One port is for the SGi interface (default: enp1s0) and one for the S1
-  interface (default: enp2s0). Note that the `agw_install.sh` script will
+  interface (default: enp2s0). Note that the `agw_install_ubuntu.sh` script will
   rename the `enp1s0` interface to `eth0`.
 
 ## Deployment
 
-### 1. Create boot USB stick and install Debian on your AGW host
+### 1. Create boot USB stick and install Ubuntu on your AGW host
 
-- Download .iso image from [Debian mirror](https://cdimage.debian.org/cdimage/archive/9.13.0/amd64/iso-cd/debian-9.13.0-amd64-netinst.iso)
+- Download Ubuntu Server 20.04 LTS .iso image from Ubuntu website
 - Create bootable usb using etcher [tutorial here](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-macos#0)
 - Boot your AGW host from USB
   (Press F11 to select boot sequence, :warning: This might be different for
   your machine). If you see 2 options to boot from USB, select the non-UEFI
   option.
 - Select “Install” option.
-- Network missing firmeware "No"
+- Network missing firmware "No"
 - Primary network interface "enp1s0"
 - Configuration.
-  * Hostname: “magma”
+  * hostname: <choose host name>. Default is "magma". You need to select unique
+    host name in case of multiple AGW in network.
   * Domain name : “”
```

Comment:
Nit: here and elsewhere, can you grep-replace all the curly quotes to be straight quotes